containers/overview/LinkItem.js

```diff
@@ -1,26 +1,42 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Link } from 'react-router-dom';
 import { Tooltip, Icon } from 'react-components';
 import { c } from 'ttag';
 
-const LinkItem = ({ route, text, permission }) => {
+const href = (route) => `${route.pathname}#${route.hash}`;
+
+const LinkItem = ({ route, text, permission, history }) => {
+    const handleClick = async (event) => {
+        event.preventDefault();
+
+        history.push(href(route));
+
+        // Let the router do the navigation
+        await null;
+
+        const el = document.querySelector(`#${route.hash}`);
+        if (el) {
+            el.scrollIntoView({ behavior: 'smooth', block: 'start' });
+        }
+    };
+
     return (
-        <Link to={route}>
+        <a onClick={handleClick} href={href(route)}>
```

Comment:
I have another proposal than this.

1. Revert this to be normal links, with a hash anchor.
2. Do the `scrollIntoView` in the `Page.js` file in a useEffect by observing the hash instead.

It still wouldn't solve the dynamic section issue, but at least it has the advantage of working also on page refresh, and it doesn't break right-click > open in a new tab (even though we don't support that atm due to session not being shared).

Reply:
You're right, it would be much easier to maintain. I'll try this.

Reply:
Working on this, I seen that the observer sidebar also do a scroll on click instead of changing the hash and react to it. Perhaps, I could change this too ?